morl_baselines/common/morl_algorithm.py

```diff
@@ -205,25 +208,41 @@ def get_config(self) -> dict:
             dict: Config
         """
 
-    def setup_wandb(self, project_name: str, experiment_name: str) -> None:
+    def register_additional_config(self, ref_point: np.ndarray, known_front: Optional[List[np.ndarray]]) -> None:
```

Comment:
Could this method simply receive a dict instead? Then we could later be used to register anything we'd like.

Reply:
Yeah I can do that, I just liked the typing haha